### PR TITLE
Libbot renamed to libbot2

### PR DESCRIPTION
**Pull request by fbudin69500**

### CODE REVIEW

File: cmake/cpack.cmake [hunk 1]
```diff
@@ -36,27 +42,30 @@ endif()
...
+set(CPACK_PACKAGE_VERSION ${PROJECT_VERSION})
 set(CPACK_PACKAGE_NAME "${PROJECT_NAME}")
 set(CPACK_PACKAGE_FILE_NAME ${CPACK_PACKAGE_NAME}_${CPACK_PACKAGE_VERSION}-${PACKAGE_RELEASE_VERSION}_${OS_TYPE_ARCH_SUFFIX})
 set(CPACK_STRIP_FILES TRUE)
 set(CPACK_SOURCE_STRIP_FILES FALSE)
 
 # Debian specific
+set(CPACK_DEBIAN_PACKAGE_NAME ${CMAKE_PROJECT_NAME})
+set(CPACK_PACKAGE_VERSION ${PROJECT_VERSION})
+set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE MACHINE_ARCH)
```
Comment (jamiesnape) [inline cmake/cpack.cmake]: `${MACHINE_ARCH}`?

File: scripts/package/linux/ubuntu/xenial/build.sh [hunk 2]
```diff
@@ -15,7 +15,7 @@ git clone https://github.com/lcm-proj/lcm
 mkdir lcm-build
 pushd lcm-build
 # Forces the package to be installed in /usr/local
-# to be consistent with libbot and the fact these packages
+# to be consistent with libbot2 and the fact these packages
 # are not "official".
 cmake -DBUILD_SHARED_LIBS:BOOL=ON \
       -DCPACK_PACKAGING_INSTALL_PREFIX:PATH=/usr/local \
```
Comment (jamiesnape) [inline scripts/package/linux/ubuntu/xenial/build.sh]: Can this install to `/opt/lcm/<version>`?

**Review by jamiesnape — APPROVED** [review]

File: cmake/cpack.cmake (hunk 1 above)
Comment (fbudin69500) [reply cmake/cpack.cmake]: Good catch.

File: scripts/package/linux/ubuntu/xenial/build.sh (hunk 2 above)
Comment (fbudin69500) [reply scripts/package/linux/ubuntu/xenial/build.sh]: Removing this here as the default location should be correct (and it is easier to handle the version in the `CMakeLists.txt` file).

**Pull request merged**